src/analysis/retail/priest/holy/modules/talents/Archon/EnergyCycleHoly.tsx

```diff
@@ -0,0 +1,129 @@
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { Options } from 'parser/core/Module';
+import Combatants from 'parser/shared/modules/Combatants';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { formatNumber } from 'common/format';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+
+import { TALENTS_PRIEST } from 'common/TALENTS';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import { TIERS } from 'game/TIERS';
+import Events, { RemoveBuffEvent, RemoveBuffStackEvent } from 'parser/core/Events';
+import { buffedBySurgeOfLight, getHealFromSurge } from '../../../normalizers/CastLinkNormalizer';
+import SPELLS from 'common/SPELLS';
+
+//https://www.warcraftlogs.com/reports/WT19GKp2VHqLarbD#fight=19``&type=auras&source=122
+
+const APOTH_MULTIPIER = 4;
+const ENERGY_CYCLE_CDR = 4;
+const LIGHT_OF_THE_NAARU_REDUCTION_PER_RANK = 0.1;
+const TWW_TIER1_2PC_CDR = 0.1;
```

Comment:
Other specs put all of their constants into a single file, might be something to consider to make it more manageable

Reply:
I have moved things to a hero tree specific file, I will move it to one main file when I do more comprehensive holy priest work.